sketch_map_tool/routes.py

```diff
@@ -188,14 +194,20 @@ def get_async_result_id(uuid: str, type_: REQUEST_TYPES):
 @app.get("/<lang>/create/results")
 @app.get("/create/results/<uuid>")
 @app.get("/<lang>/create/results/<uuid>")
-def create_results_get(lang="en", uuid: str | None = None) -> Response | str:
+@app.get("/create/results/<uuid>/<bbox>")
+@app.get("/<lang>/create/results/<uuid>/<bbox>")
+def create_results_get(
+    lang="en", uuid: str | None = None, bbox: str = ""
+) -> Response | str:
     if uuid is None:
         return redirect(url_for("create", lang=lang))
+    if not re.match(r"^[0-9,.]*$", bbox):
+        bbox = ""
```

Comment:
Could you expand a bit on this line?

Shouldn't always a bounding box be given?

What exactly is validated?

Reply:
If there are only numbers `.` and `,` in the input. Since this is fed directly into the template, it should be validated somehow. Doesn't need to be `re`, could also be a more advanced validation algorithm.

Reply:
Alternate solution: 301c56d